packages/@unimodules/core/ios/UMCore.podspec

```diff
@@ -10,12 +10,13 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, "10.0"
+  s.platform       = :ios, '10.0'
   s.source         = { :git => "https://github.com/expo/expo.git" }
-  s.source_files   = 'UMCore/**/*.{h,m}'
-  s.preserve_paths = 'UMCore/**/*.{h,m}'
-  s.requires_arc   = true
 
+  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework")
+    s.source_files = "#{s.name}/**/*.h"
+    s.vendored_frameworks = "#{s.name}.xcframework"
+  else
+    s.source_files = "#{s.name}/**/*.{h,m}"
+  end
```

Comment:
Question/idea: don't we want to have a function in `UMCore` (or some similar package) that would do that and simply invoke it here? 🤔